udata/translations/fr/LC_MESSAGES/udata.po

```diff
@@ -882,7 +882,7 @@ msgstr "Une discussion a été clôturée"
 #: udata/core/discussions/mails.py:53
 #, python-format
 msgid "The discussion you participated in on the %(type)s %(object)s has been closed by %(user_or_org)s."
-msgstr "La discussion à laquelle vous avez participé sur l' %(type)s %(object)s a été clôturée par %(user_or_org)s."
+msgstr "La discussion à laquelle vous avez participé sur %(type)s %(object)s a été clôturée par %(user_or_org)s."
```

Comment:
I've updated this translation. It's not perfect, but I think at the moment we don't juste the determiner since it's not always the same in French